client/components/Login.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+export default class Login extends React.Component {
+  state = {
+    username: '',
+    password: ''
+  }
+
+  handleChange = e => {
+    this.setState({
+
+      [e.target.name]: e.target.value
+    })
+  }
+
+ handleClick = e => {
+   e.preventDefault()
+ }
+
+ render () {
+   return (
+     <React.Fragment>
+       <h1>Login</h1>
+
+       <p>Username:</p>
+       <input name='username'
```

Comment:
I believe this should be `email` instead of `username`. The fields to register are firstName, lastName, email and password. Email being the unique identifier to log in.

Reply:
Hi Emily, the Live Demo of the prototype shows username in login page, should I change it to email instead?

Reply:
Yes please! Chris' demo will need updating - as we have updated how we log in :)